fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -436,15 +441,17 @@ public Response createOrReplaceObjectRdf(
                         .toString(),
                     model);
             } else {
-                createResourceService.perform(transaction.getId(), getUserPrincipal(), fedoraId, null, false, links,
+                fedoraId = createResourceService.perform(transaction.getId(), getUserPrincipal(), fedoraId, null,
```

Comment:
Should probably have this on two lines rather than going to three, or one line per parameter.

Reply:
will fix

Reply:
This is fixed.